src/ensembl/production/metadata/updater/core.py

```diff
@@ -58,496 +62,411 @@ def process_core(self, **kwargs):
+
+                # Test to see if the seq_name follows accession standards (99% of sequences)
+                if re.match(r'^[a-zA-Z]+\d+\.\d+', seq_region_name):
+                    # If so assign it to accession
+                    accession = seq_region_name
+                    if not synonym:
+                        # If it doesn't have any synonyms the accession is the name.
+                        accession_info[accession]["names"].add(accession)
+                    else:
+                        accession_info[accession]["names"].add(synonym)
+                else:
+                    # For named sequences like chr1
+                    name = seq_region_name
+                    if re.match(r'^[a-zA-Z]+\d+\.\d+', synonym):
+                        accession = synonym
+                        accession_info[accession]["names"].add(name)
+                    else:
+                        accession = name  # In case synonym doesn't match the pattern, use the name as the accession
+                        accession_info[accession]["names"].add(synonym if synonym else name)
+
+                # Save the sequence location, length, and chromosomal flag.
+                location_mapping = {
+                    'nuclear_chromosome': 'SO:0000738',
+                    'mitochondrial_chromosome': 'SO:0000737',
+                    'chloroplast_chromosome': 'SO:0000745',
+                    None: 'SO:0000738',
+                }
+
+                try:
+                    sequence_location = location_mapping[location]
+                except KeyError:
+                    raise Exception('Error with sequence location: {} is not a valid type'.format(location))
+
+                # Test if chromosomal:
+                if coord_system_name == "chromosome":
+                    chromosomal = 1
+                else:
+                    chromosomal = 0
+
+                # Assign the values to the dictionary
+                accession_info[accession]["location"] = sequence_location
+                accession_info[accession]["chromosomal"] = chromosomal
+                accession_info[accession]["length"] = seq_region_length
+
+            # Now, create AssemblySequence objects for each unique accession.
+            for accession, info in accession_info.items():
+                # Combine all unique names with ";". If a name appears in multiple sequences with the same accession,
+                name = ";".join(info["names"])
+
+                # Create an AssemblySequence object.
+                assembly_sequence = AssemblySequence(
+                    name=name,
+                    assembly=assembly,
+                    accession=accession,
+                    chromosomal=info["chromosomal"],
+                    length=info["length"],
+                    sequence_location=info["location"],
+                    # sequence_checksum="", Not implemented
```

Comment:
Checksums will be in the core databases now so we can accept and load this right away correct?

Reply:
Should we remove the columns from the schema ?

Reply:
After more discussions yesterday the answer is no. Reason - the genome browser may use these data to retrieve CHECKSUMS in order to request sequence from refget proxy.

Reply:
Also, the metadata API will eventually be the source of truth of these CHECKSUMS for programmatic access. We definitely need it now

Reply:
Will not do, as the checksums will be generated post-handover